qnexus/client/__init__.py

```diff
@@ -98,3 +104,13 @@ def get_nexus_client(reload: bool = False) -> httpx.Client:
             verify=CONFIG.httpx_verify,
         )
     return _nexus_client
+
+
+def _print_sunset_header_warning(
+    request: httpx.Request, response: httpx.Response
+) -> None:
+    sunset_header = response.headers.get("sunset") or response.headers.get("Sunset")
```

Comment:
Headers are case insensitive so getting sunset should be enough
```suggestion
    sunset_header = response.headers.get("sunset")
```

Reply:
Ok, cheers